app/src/main/AndroidManifest.xml

```diff
@@ -53,7 +53,7 @@
 
         <meta-data
             android:name="firebase_performance_logcat_enabled"
-            android:value="true" />
```

Comment:
There is a debug AndroidManifest file where we can override this option too, but I prefer your approach.

Reply:
@aeramli I saw it, this approach is more flexible and could support multi build types and/or flavours.